util/dvsim/LocalLauncher.py

```diff
@@ -43,29 +43,48 @@ def _do_launch(self):
 
         self._dump_env_vars(exports)
 
-        try:
-            f = open(self.deploy.get_log_path(),
-                     "w",
-                     encoding="UTF-8",
-                     errors="surrogateescape")
-            f.write("[Executing]:\n{}\n\n".format(self.deploy.cmd))
-            f.flush()
-            timeout_mins = self.deploy.get_timeout_mins()
-            if timeout_mins:
-                self.timeout_secs = timeout_mins * 60
-            else:
-                self.timeout_secs = None
-            self.process = subprocess.Popen(shlex.split(self.deploy.cmd),
-                                            bufsize=4096,
-                                            universal_newlines=True,
-                                            stdout=f,
-                                            stderr=f,
-                                            env=exports)
-        except subprocess.SubprocessError as e:
-            raise LauncherError('IO Error: {}\nSee {}'.format(
-                e, self.deploy.get_log_path()))
-        finally:
-            self._close_process()
+        if not self.deploy.sim_cfg.interactive:
+            try:
+                f = open(self.deploy.get_log_path(),
+                         "w",
+                         encoding="UTF-8",
+                         errors="surrogateescape")
+                f.write("[Executing]:\n{}\n\n".format(self.deploy.cmd))
+                f.flush()
+                timeout_mins = self.deploy.get_timeout_mins()
+                if timeout_mins:
+                    self.timeout_secs = timeout_mins * 60
+                else:
+                    self.timeout_secs = None
+                self.process = subprocess.Popen(shlex.split(self.deploy.cmd),
+                                                bufsize=4096,
+                                                universal_newlines=True,
+                                                stdout=f,
+                                                stderr=f,
+                                                env=exports)
+            except subprocess.SubprocessError as e:
+                raise LauncherError('IO Error: {}\nSee {}'.format(
+                    e, self.deploy.get_log_path()))
+            finally:
+                self._close_process()
+        else:
+                # Interactive: Set RUN_INTERACTIVE to 1
```

Comment:
Is this specific to DC / Primetime? If so, then I am not sure if this is the right place to put this export. The `Launcher` needs to remain agnostic to individual tool requirements.